src/formatter-config.ts

```diff
@@ -14,18 +14,8 @@ const defaultCljfmtContent =
 const LSP_CONFIG_KEY = 'CLOJURE-LSP';
 let lspFormatConfig: string | undefined;
 
-function cljfmtOptionsFromString(cljfmt: string) {
-  const options = cljsLib.cljfmtOptionsFromString(cljfmt);
-  return options.error
-    ? options
-    : {
-        ...options,
-        indents: options['indents'],
-      };
-}
-
 function configuration(workspaceConfig: vscode.WorkspaceConfiguration, cljfmt: string) {
-  const cljfmtOptions = cljfmtOptionsFromString(cljfmt);
+  const cljfmtOptions = cljsLib.cljfmtOptionsFromString(cljfmt);
```

Comment:
Does this change mean that in case of error, we would get the defaultIndents rather than just the indents from options?

Looking at
```
(defn- read-cljfmt
  [s]
  (try
    (as-> s $
      (parse-clj-edn $))
    (catch js/Error e
      (merge default-fmt
             {:error (.-message e)
              :indents cljfmt/default-indents}))))
```

in formatter.cljs

Reply:
Yes. “just the indents from options” are not available if the parsing fails.